src/patternfly/components/Drawer/drawer.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-drawer {
```

Comment:
when the panel is closed, should it still be visible?

<img width="601" alt="Screen Shot 2019-07-17 at 3 15 20 PM" src="https://user-images.githubusercontent.com/20118816/61404604-c7670b00-a8a5-11e9-97c5-822e7e9b16a9.png">

Reply:
That's just a workspace thing, the examples have padding so it's showing in the padding. If you remove the padding, it's outside of `.pf-c-drawer`, and `overflow-x` is `hidden`

<img width="806" alt="Screen Shot 2019-07-17 at 4 16 57 PM" src="https://user-images.githubusercontent.com/35148959/61412391-5d9f2f00-a8ae-11e9-8bda-04c89be528d0.png">